client/css/style.css

```diff
@@ -1113,14 +1115,22 @@ kbd {
 
 #chat .toggle-content img {
 	max-width: 100%;
-	max-height: 250px;
+	max-height: 128px;
 	display: block;
-	margin: 2px 0;
 }
 
 #chat .toggle-content .thumb {
-	max-height: 110px;
-	max-width: 210px;
+	float: left;
```

Comment:
I have not. Floats are very much designed for this kind of usage, and it works just fine like this sooo I'm tempted to call it good enough and go on with my life :) (also it was like this initially, I did not try to be creative)
All I missing a clear issue that flexbox addresses here?